paasta_tools/cli/schemas/chronos_schema.json

```diff
@@ -86,6 +86,10 @@
                 },
                 "uniqueItems": true
             },
+            "networking_mode": {
+                "enum": [ "BRIDGE", "HOST" ],
```

Comment:
I don't want this to be restrictive, as other users of this software might want to do custom named networking and stuff. Just say it is a string